[FW][FIX] l10n_ar_txt_sire: Fix arquitecture of the payment view

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/odoo-argentina-ee#619

rov-adhoc committed:
[FIX] l10n_ar_txt_sire: Fix arquitecture of the payment view

X-original-commit: 2aca73be9666656a2bb6dc44d3d97dea1081b331

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/620).

fw-bot-adhoc commented:
@rov-adhoc @maq-adhoc cherrypicking of pull request ingadhoc/odoo-argentina-ee#619 failed.

stdout:
```
CONFLICT (modify/delete): l10n_ar_txt_sire/views/account_payment_view.xml deleted in HEAD and modified in f86b3f1 ([FIX] l10n_ar_txt_sire: Fix arquitecture of the payment view). Version f86b3f1 ([FIX] l10n_ar_txt_sire: Fix arquitecture of the payment view) of l10n_ar_txt_sire/views/account_payment_view.xml left in tree.
CONFLICT (modify/delete): l10n_ar_txt_sire/__manifest__.py deleted in HEAD and modified in f86b3f1 ([FIX] l10n_ar_txt_sire: Fix arquitecture of the payment view). Version f86b3f1 ([FIX] l10n_ar_txt_sire: Fix arquitecture of the payment view) of l10n_ar_txt_sire/__manifest__.py left in tree.

```

stderr:
```
13:22:45.164684 git.c:444               trace: built-in: git cherry-pick f86b3f12050e33db3eabfd76c1af5e155580ce03
error: could not apply f86b3f1... [FIX] l10n_ar_txt_sire: Fix arquitecture of the payment view
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

rov-adhoc commented:
No se requieren en 17 ni en 18
